ecl/hql/hqlgram.y

```diff
@@ -4776,8 +4776,24 @@ fieldAttr
                         {
                             $$.setExpr(createExprAttribute(xmlDefaultAtom, $3.getExpr()), $1);
                         }
-    | DEFAULT '(' constExpression ')'
+    | DEFAULT '(' goodObject ')'
                         {
+                            IHqlExpression * arg = $3.queryExpr();
+                            if (arg->getOperator() == no_temptable && arg->queryChild(0)->queryType()==nullptr)
```

Comment:
Why is this line needed?  At the least it needs a comment.

Reply:
This code is no longer necessary as it is now handled by createTempTableAssign